ifopt_core/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ install(
   EXPORT  ${LIB_CORE}-targets
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Comment:
I would also remove the definition of the variable IFOPT_INSTALL_BINDIR above since it is no longer used.

Reply:
Yea it likes only the examples. Is there any reason not to install the examples into ${CMAKE_INSTALL_BINDIR}?

Reply:
Well, I didn't see any reason it should change from the current `lib/${PROJECT_NAME}`. It's the ROS convention and it has the benefit of avoiding potential name collisions with other packages. See also this [ROS answers question](https://answers.ros.org/question/367854/proper-runtime-executable-install-location/) I asked on this topic.